www/templates/www/banktransaction.html

```diff
@@ -1,8 +1,5 @@
-{% extends "www/base.html" %}
-{% load i18n %}
-{% load bootstrap4 %}
-{% block content %}
-{% load verbose_names %}
+{% extends "www/base.html" %} {% load i18n %} {% load bootstrap4 %} {% block
+content %} {% load verbose_names %}
```

Comment:
Can we keep these on separate lines? (I'm guessing your IDE did some autoformatting here)